pkg/kademlia/kademlia.go

```diff
@@ -309,6 +309,11 @@ func (k *Kad) manage() {
 								}
 								k.logger.Debugf("peer not reachable from kademlia %s: %v", bzzAddr.String(), err)
 								k.logger.Warningf("peer not reachable when attempting to connect")
+
+								k.waitNextMu.Lock()
+								k.waitNext[peer.String()] = retryInfo{tryAfter: time.Now().Add(shortRetry)}
```

Comment:
maybe we can add this at the end of the loop because we also have a wait in the case that there is no error. this way we wont repeat the same wait logic